spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -171,16 +171,34 @@
       end
     end
 
-    context 'PhoneQuestion page' do
-      before do
+    context 'with PhoneQuestion page enabled' do
+      before(:each) do
```

Comment:
i see why you added this before block to reduce duplicate code 👍🏿 .  my understanding is  that this phone question page is temporary and will be taken down shortly?  while it is somewhat redundant (creating an authed user and completing agreement step)should we consider not refactor the rate limit test which will allow us to simply delete the phone question test once our a/b test is complete? 🤔